apps/exevo-pan/src/modules/BossTracker/components/BossGrid/utils.ts

```diff
@@ -75,3 +76,18 @@ export const listBy = {
   rook,
   pinned,
 }
+
+export const prioritizePremium: BossLister = (list) => {
+  const premium: BossStats[] = []
+  const free: BossStats[] = []
+
+  list.forEach((bossStat) => {
+    if (premiumBosses.set.has(bossStat.name)) {
+      premium.push(bossStat)
+    } else {
+      free.push(bossStat)
+    }
+  })
+
+  return premium.concat(free)
+}
```

Comment:
Why are premium bosses prioritized? As a user, I would prefer all bosses to show up based on their % chance. It's fine if you want to highlight premium bosses in a special way (and the styling already accomplishes that IMHO) but it's not helpful to push the premium bosses to the start if the list even if there's no chance of them spawning on my server today.

Here's what I'm seeing for my list of Favorites for my server:

![image](https://user-images.githubusercontent.com/81942/207165781-b914d7f2-51d6-4586-b12d-e46097c2fc43.png)

None of the premium bosses have any chance of spawning today, yet they take up space at the start of the list.

Would you be willing to reconsider this? The code change would be simple -- just don't use `prioritizePremium` at all.

Reply:
Moved to https://github.com/xandjiji/exevo-pan/issues/150